test/basic.html

```diff
@@ -22,6 +22,17 @@
   <link rel="import" href="../paper-radio-button.html">
 
 </head>
+<style is="custom-style">
+  paper-radio-button.large {
+    --paper-radio-button-size: 48px;
+  }
+  paper-radio-button.medium {
+    --paper-radio-button-size: 24px;
+  }
+  paper-radio-button.small {
+    --paper-radio-button-size: 12px;
+  }
+</style>
```

Comment:
Oh, there's a way to get around the height part of this which is used in paper-checkbox ([here](https://github.com/PolymerElements/paper-checkbox/blob/master/paper-checkbox.html#L62), and [here](https://github.com/PolymerElements/paper-checkbox/blob/master/paper-checkbox.html#L174)), add `line-height: 0;` to the `:host` rule and `line-height: normal;` to `#radioLabel`. This way, the line-height of the label will only affect the container's height when there's actually text in it.